content/docs/command-reference/remote/modify.md

```diff
@@ -277,6 +277,9 @@ $ dvc remote modify myremote endpointurl \
                              https://nyc3.digitaloceanspaces.com
 ```
 
+Please refer to the "Amazon S3" section for a list of complete S3 variables;
+whether the variable is useful depends on your actual S3 backend.
+
```

Comment:
Good note! But this is already mentioned in the previous paragraph: "Besides that, any settings that are available for Amazon S3 (see previous section) may be available for S3 compatible storage."
```suggestion
```